addon/instance-initializers/hot-loader-livereload-plugin.js

```diff
@@ -41,7 +46,12 @@ export function initialize(appInstance) {
   if (!window.LiveReload) {
     return;
   }
-  const Plugin = createPlugin(appInstance.base.name, appInstance.lookup('service:hot-reload'));
+  let appName = appInstance.base.name;
+  if (appName === 'ember-cli-hot-loader') {
```

Comment:
We need a better way to detect if we're running an addon's dummy app, for now, this only works with apps and with this addon. I created https://github.com/toranb/ember-cli-hot-loader/issues/20 as a reminder to fix it.